ch02.adoc

```diff
@@ -51,6 +51,7 @@ By the word _letters_ we mean the standard ASCII letters uppercase `A` to `Z` an
 By the word _digits_ we mean the standard ASCII digits `0` to `9`, and similarly _underscores_ means the standard ASCII underscore `_`.
 Note that this is in conformance with the COARDS conventions, but is more restrictive than the netCDF interface which allows almost all Unicode characters encoded as multibyte UTF-8 characters (link:$$https://docs.unidata.ucar.edu/nug/current/file_format_specifications.html$$[NUG Appendix B]).
 The netCDF interface also allows leading underscores in names, but the NUG states that this is reserved for system use.
+ASCII period (.) and ASCII hyphen (-) are also allowed in attribute names only.
 
```

Comment:
Perhaps backticks "`"

```ASCII period (.) and ASCII hyphen (-) ... ```

to
``` ASCII period (`.`) and ASCII hyphen (`-`) ... ```

to keep it consistent with previous text?

Reply:
I respectfully disagree with your requested changes.  The text is clear as I have drafted.